app/src/main/java/mozilla/lockbox/action/TelemetryAction.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.action
+
+import mozilla.lockbox.flux.Action
+import org.mozilla.telemetry.event.TelemetryEvent
+
+open class TelemetryAction(
+    val eventMethod: TelemetryEventMethod,
```

Comment:
These will probably all need to be `open` as well so that they can be overridden by implementing `Action`s. Having a sample case would help shake out these problems.

Reply:
It's the constructor, so why can't implementing actions apply it then?